dom/test/browser/events.js

```diff
@@ -340,6 +340,52 @@ describe('DOMSource.events()', function () {
     dispose = run();
   });
 
+  it('should catch bubbling events in a DocumentFragment (and shadowRoot)', function (done) {
+    function app() {
+      return {
+        DOM: Rx.Observable.of(div([
+          div('.clickable', 'Hello')
+        ]))
+      }
+    }
+
+    function click(el) {
+      const ev = document.createEvent(`MouseEvent`)
+      ev.initMouseEvent(
+        `click`,
+        true /* bubble */, true /* cancelable */,
+        window, null,
+        0, 0, 0, 0, /* coordinates */
+        false, false, false, false, /* modifier keys */
+        0 /*left*/, null
+      )
+      el.dispatchEvent(ev)
+    }
```

Comment:
We use just `element.click()` in other tests, so I think you can remove this helper function.